app/routes/socialEvents.tsx

```diff
@@ -0,0 +1,15 @@
+import EventCard from '~/components/common/eventCard';
+
+export default function socialEvents() {
+  return (
+    <>
+      <EventCard
+        date={30}
+        month="AUG"
+        time="12AM to 1:30PM"
```

Comment:
Wouldn't it be good if we separate as fromTime and toTime here?

Reply:
+1

Reply:
Sounds good, updating it 
Thanks!

Reply:
Done, updated. 
Thanks